config/activity_labels/all_recipe_labels.yaml

```diff
@@ -10,216 +10,209 @@ labels:
   # - repeat: Optional field that indicated that the item must be repeated X times
```

Comment:
We can probably remove the `depends` and `repeat` keys. They seem like they were for task step configs more thanctivity classification?

Reply:
They were used to generate the out of order steps for the annotation team to create their data with. Idk if we'd want that on the tasks since they are higher level than the activities - we'd lose a lot of mixing of steps

Reply:
There has just been a lot of concept mixing between activities and tasks/steps. I'm not clear on what the mixed legacy is any more.

Reply:
Can I leave it for now until the tools that use the depends and repeat keys are updated?